### ISSUE TITLE
Loss Function Identification

### ISSUE THREAD

**Issue by shakhal350** [shakhal350/VitalSign-Capstone-2023#17]
Determine the loss function to be used for the neural network
